cmd/cryptogen/main.go

```diff
@@ -265,6 +280,40 @@ func getConfig() (*Config, error) {
 		return nil, fmt.Errorf("Error Unmarshaling YAML: %s", err)
 	}
 
+	for idx := range config.OrdererOrgs {
+		if config.OrdererOrgs[idx].CACertExpiry == 0*time.Second {
```

Comment:
Please use `time.Duration(0)` instead of `0*time.Second` since we're looking for a zero duration, not zero seconds.

We have also been badly burned by people assuming the units of a duration defaults to seconds instead of nanoseconds. If we see a value that's less than our 5 minute time shift window, we should really exit with a nice error message highlighting the problem.